crypto/ec/ec_key.c

```diff
@@ -552,6 +579,21 @@ int EC_KEY_set_private_key(EC_KEY *key, const BIGNUM *priv_key)
         return 0;
     BN_clear_free(key->priv_key);
     key->priv_key = BN_dup(priv_key);
+    key->dirty_cnt++;
+    if (key->priv_key != NULL) {
+        int group_top;
+        const BIGNUM *order;
+
+        BN_set_flags(key->priv_key, BN_FLG_CONSTTIME);
+
+        order = EC_GROUP_get0_order(key->group);
```

Comment:
A comment here would be nice..

Reply:
This is change comes from one of the hardening commits that are included in this PR because I stumbled upon them while working on the internals.

I plan to open a PR to backport these hardening commits in 1.1.1.

For some reason, maybe thinking it would have been easier to cherry-pick later, I deliberately avoided putting a comment in the code and limited it to a comment in the commit message.

But you are right, I reconsidered, so I will be including a comment in the code as well!